Resources/Prototypes/Species/arachnid.yml

```diff
@@ -18,6 +18,8 @@
 - type: speciesBaseSprites
   id: MobArachnidSprites
   sprites:
+    TailBehind: MobHumanoidAnyMarking
+    TailOversuit: MobHumanoidAnyMarking
```

Comment:
Okay GitHub is starting to cry with the size of this PR, but it looks like floof comments are needed for most all prototype changes to upstream and _DV species changes; I'd just go through and check all of them.